app/src/main/java/com/rentminder/Icon&Text.kt

```diff
@@ -86,6 +79,40 @@ fun WiFiIconText() {
     }
 }
 
+@Composable
+fun TotalIconText() {
+    Row(verticalAlignment = Alignment.CenterVertically){
+        Icon(painterResource(
+            id = R.drawable.outline_wifi_24
+        ),
+            contentDescription = "Total Icon",
+            modifier = Modifier
+                .size(30.dp)
+        )
+        Text(
+            text = "Total",
+            fontSize = (26.sp),
+        )
+    }
+}
+
+@Composable
+fun DividedIconText() {
+    Row(verticalAlignment = Alignment.CenterVertically){
+        Icon(painterResource(
+            id = R.drawable.outline_wifi_24
+        ),
+            contentDescription = "Divided Icon",
+            modifier = Modifier
+                .size(30.dp)
+        )
+        Text(
+            text = "Divided",
+            fontSize = (26.sp),
+        )
+    }
+}
```

Comment:
This looks like an incomplete feature.  I don't see an onClick function.  
Are you using these anywhere?   If not used, this will be an unnecessary distraction to the user, and clutter in code.

Recommendation: if not used, remove it, or keep it in a separate development branch until it is complete.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.